client/signup/steps/plans/index.jsx

```diff
@@ -102,13 +103,23 @@ module.exports = React.createClass( {
 
 	plansSelection: function() {
 		let headerText = this.translate( 'Pick a plan that\'s right for you.' ),
+			subHeaderText;
+
+		if ( config.isEnabled( 'upgrades/free-trials' ) ) {
+			subHeaderText = this.translate(
+				'Try WordPress.com Premium or Business free for 14 days, no credit card required.'
+			);
+		} else {
 			subHeaderText = this.translate(
 				'Not sure which plan to choose? Take a look at our {{a}}plan comparison chart{{/a}}.',
-				{ components: { a: <a
-					href={ this.comparePlansUrl() }
-					onClick={ this.handleComparePlansLinkClick.bind( null, 'header' ) } /> } }
+				{ 
+					components: { a: <a
+						href={ this.comparePlansUrl() }
+						onClick={ this.handleComparePlansLinkClick.bind( null, 'header' ) } /> }
+				}
 			);
 
```

Comment:
Minor code style issue here. You should move that empty line after the closing `}`